app/views/posts/index.html.erb

```diff
@@ -1 +1,44 @@
-<h1>This is the posts file for testing</h1>
\ No newline at end of file
+<div style="margin: 20px 40px;">
+  <div class="card mb-3" style="border: none;">
+    <div class="row g-0" style="gap: 1.5%;">
+      <div class="col-md-4" style="width: 70px;">
+        <img src ='<%= @user.photo %>' class="img-fluid rounded" alt="User Photo">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body" style="border: 1px solid black; margin-top: 5px; width: 1345px;">
+          <h5 class="card-title"><%= @user.name %></h5>
+          <p class="card-text"><small class="text-muted">
+            Number of posts: <%= @user.posts_counter %></small>
+          </p>
+        </div>
+      </div>
+    </div>
```

Comment:
👏 Great work on your project! Here's a suggestion to enhance your codebase:

- [ ] **DRY Up User Details Display:** There is repetitive code for displaying user details in `user#index`, `user#show`, and `post#index` erb files. To adhere to the DRY (Don't Repeat Yourself) principle, it's recommended to eliminate redundancy and maintain a clean codebase. One effective solution is to use [partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials).

   Create a partial file, for example, `_user.html.erb`:

   ```erb
   <!-- _user.html.erb -->

   <div class="card mb-3" style="border: none;">
     <div class="row g-0" style="gap: 1.5%;">
       <div class="col-md-4" style="width: 70px;">
         <img src='<%= user.photo %>' class="img-fluid rounded" alt="User Photo">
       </div>
       <div class="col-md-8">
         <div class="card-body" style="border: 1px solid black; margin-top: 5px; width: 1345px;">
           <h5 class="card-title"><%= user.name %></h5>
           <p class="card-text">
             <small class="text-muted">Number of posts: <%= user.posts_counter %></small>
           </p>
         </div>
       </div>
     </div>
   </div>
   ```

   Then, in your view, render this partial:

   ```erb
   <!-- app/views/posts/index.html.erb -->
   <%= render partial: 'user', locals: { user: @user } %>
   ```

   ```erb
   <!-- app/views/users/show.html.erb -->
   <%= render partial: 'user', locals: { user: @user } %>
   ```
**_Note: Perhaps the example code provided may not be directly applicable to your codebase. You can optimize it as needed._**
   By using partials, you'll improve maintainability and adhere to best practices. Keep up the excellent work! 🚀👩‍💻